### DISCUSSION TITLE
Recently Used Colors Auto-Added to Palette

### DISCUSSION THREAD

**Discussion by fagerburg** [Orama-Interactive/Pixelorama#859] [Ideas]
Thanks for this incredibly awesome tool! It would be really convenient (for me at least) if there was an option to enable automatically adding the colors you draw on the canvas with to a palette without having to create a new palette from current sprite/selection or having to click the plus button each time to add a color. I don't believe this currently exists but if it does feel free to close this issue.

**Comment by OverloadedOrama** [comment]
I implemented a similar feature in https://github.com/Orama-Interactive/Pixelorama/commit/519fa777910176525774ee520536cc743e4f134c. It doesn't interact with the palette system, but up to 9 recent colors are now shown below the color picker. I'll go ahead and close this as implemented for now, but if you have something else in mind, feel free to re-open.
